app/views/permissions/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class='container mt-5'>
```

Comment:
try to change the file/controller from permission to USER

Reply:
replaced the controller with the existing one